### PR TITLE
[IMP] l10n_se: Update taxes and accounts

### PR DESCRIPTION
**Pull request by JulienAlardot**
[IMP] l10n_se: Update taxes and accounts

- Switch the taxes and account names to english by default
- Changes the tax names to match new taxonomy
- Account changes following 2023 BAS update
   ° Add 2024 accounts
   ° Remove 2015 accounts
   ° Several changes in names as codes

Task-3052677
Task-3122530
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by gawa-odoo — COMMENTED** [review]
I would prefer keeping quotes around names, as they existed before. It keeps it consistent when you have commas in the name and you need it.
It also makes it clearer to read when it is not formatted in columns (like for the taxes in the diff, where it is really hard to see the difference between columns)

File: addons/l10n_se/data/template/account.account-se.csv [hunk 1]
```diff
@@ -1,330 +1,329 @@
...
-"a2090","2090","Fritt eget kapital","equity","l10n_se.account_tag_39","False"
-"a2091","2091","Balanserad vinst eller förlust","equity","l10n_se.account_tag_12","False"
-"a2098","2098","Vinst eller förlust från föregående år","equity","l10n_se.account_tag_11","False"
-"a2099","2099","Årets resultat","equity","l10n_se.account_tag_17","False"
-"a2120","2120","Periodiseringsfond 2020","equity","l10n_se.account_tag_18","False"
-"a2121","2121","Periodiseringsfond 2021","equity","l10n_se.account_tag_18","False"
-"a2122","2122","Periodiseringsfond 2022","equity","l10n_se.account_tag_18","False"
-"a2123","2123","Periodiseringsfond 2023","equity","l10n_se.account_tag_18","False"
-"a2124","2124","Periodiseringsfond 2024","equity","l10n_se.account_tag_18","False"
-"a2125","2125","Periodiseringsfond 2015","equity","l10n_se.account_tag_18","False"
```
[the remark this answers is not in the document]
Comment (JulienAlardot) [reply addons/l10n_se/data/template/account.account-se.csv:82]: True, and yes that account has been deleted

File: addons/l10n_se/data/template/account.account-se_K2.csv [hunk 2]
```diff
@@ -1,857 +1,857 @@
...
+"a3612","3612",Sale of scrap metal,"income","l10n_se.account_tag_31","False",Försäljning av skrot
+"a3613","3613",Sales of consumables,"income","l10n_se.account_tag_31","False",Försäljning av förbrukningsmaterial
+"a3619","3619",Sales of other materials,"income","l10n_se.account_tag_31","False",Försäljning av övrigt material
+"a3620","3620",Temporary hiring of personnel,"income","l10n_se.account_tag_31","False",Tillfällig uthyrning av personal
+"a3630","3630",Temporary use of transport equipment,"income","l10n_se.account_tag_31","False",Tillfällig av transportmedel
+"a3670","3670",Income from securities,"income","l10n_se.account_tag_31","False",Intäkter från värdepapper
+"a3671","3671",Sales of securities,"income","l10n_se.account_tag_31","False",Försäljning av värdepapper
+"a3672","3672",Dividends from securities,"income","l10n_se.account_tag_31","False",Utdelning från värdepapper
+"a3679","3679",other income from securities,"income","l10n_se.account_tag_31","False",övriga intäkter från värdepapper
+"a3680","3680",Management fees,"income","l10n_se.account_tag_31","False",Management fees
```
[the remark this answers is not in the document]
Comment (JulienAlardot) [reply addons/l10n_se/data/template/account.account-se_K2.csv:373]: image
Yes, I was as surprised as you but apparently it's a term borrowed from english

File: addons/l10n_se/data/template/account.account-se.csv (hunk 1 above)
[the remark this answers is not in the document]
Comment (JulienAlardot) [reply addons/l10n_se/data/template/account.account-se.csv:82]: Do you want an prefer an list in the commit or just something like:
- Account changes following 2023 BAS update
   ° Add 2024 accounts
   ° Remove 2015 accounts
   ° Several changes in names as codes

[the remark this answers is not in the document]
Comment (gawa-odoo) [reply addons/l10n_se/data/template/account.account-se.csv:82]: Keep it simple, just say broadly what you changed, but just so we're not surprised to see that you changed accounts when you just "translated" them

**Comment by qdp-odoo** [comment]
@robodoo r+

**Pull request closed without merging**